content/docs/glossary/muxed-accounts.mdx

```diff
@@ -59,6 +59,7 @@ Not all operations can be used with muxed accounts. For example, you cannot set
     * [`PathPaymentStrictSend`](../start/list-of-operations.mdx#path-payment-strict-send), and
     * [`PathPaymentStrictReceive`](../start/list-of-operations.mdx#path-payment-strict-receive)
   - the destination of an [`AccountMerge`](../start/list-of-operations.mdx#account-merge), and
+  - the "target" of a [`Clawback`](../start/list-of-operations.mdx#clawback) operation (i.e. the `from` field) 
```

Comment:
💡 We don't use double quotes in this fashion elsewhere in the docs, so I'd avoid using it here unless you think it adds value.
```suggestion
  - the target of a [`Clawback`](../start/list-of-operations.mdx#clawback) operation (i.e. the `from` field) 
```

Reply:
Consistency :+1: - alleviated.